src/server/jobs/update-bcast.js

```diff
@@ -262,11 +262,16 @@ async function main() {
 
           // For each listing, push new database record into array
           for (const mediaEntry of mediaEntries) {
+            const weekNum = await registrar.getWeekNumber(
```

Comment:
@chern Log if you get null for getWeekNumber(). A null value is a sign of an exception, so we want to note that.

Reply:
@rlorenzo Oops, I overlooked this… I'll add this right now